File: L-35.c

#include "stdio.h"
#pragma warning(disable:4996)
#include "stdlib.h"
#include "string.h"

int main()
{
	char name[1000][11] = { "" };
	int num=0;
	while (1)
	{
		scanf("%s",name[num++]);
		if (strcmp(".",name[num-1])==0)
		{
			break;
		}
	}
	if (num<=2)
	{
		printf("Momo... No one is for you ...\n");
	}
	else if (num<=14)
	{
		printf("%s is the only one for you...",name[1]);
	}
	else
	{
		printf("%s and %s are inviting you to dinner...\n",name[1],name[13]);
	}
	
	return 0;
}

File: L-35.py

lis=[]
while 1:
    io=input()
    if io=='.':
        break
    lis.append(io)

l=len(lis)

if  l<2:
    print('Momo... No one is for you ...')
elif l>=2 and l<14:
    print(f'{lis[1]} is the only one for you...')
else:
    print(f'{lis[1]} and {lis[13]} are inviting you to dinner...')

